KarhooUISDK/Screens/BookingScreen/KarhooBookingViewController.swift

```diff
@@ -125,9 +125,21 @@ final class KarhooBookingViewController: UIViewController, BookingView {
 
     private func setupQuoteListPanel() {
         let mainPanelVC = FloatingPanelController()
+        
+        let appearance = SurfaceAppearance()
```

Comment:
does this massively change appearance ? able to attach a screenshot to the PR if so?

Reply:
Nope, didn't change the appearance, this is mainly for the shadow. The Quote view still looks like it did before.